lib/modules/measure/MeasureService.ts

```diff
@@ -106,7 +114,7 @@ export class MeasureService {
         asset,
         measurements,
         payloadUuids
-      );
+      ).sort((measureA, measureB) => measureA.measuredAt - measureB.measuredAt);
```

Comment:
Can you move the `sort` in the `buildMeasures` method ?

Reply:
done